homeassistant/components/airly/air_quality.py

```diff
@@ -0,0 +1,207 @@
+"""Support for the Airly service."""
+import asyncio
+import logging
+from datetime import timedelta
+
+import async_timeout
+from airly import Airly
+from airly.exceptions import AirlyError
+
+from homeassistant.const import (
+    CONF_API_KEY,
+    CONF_LATITUDE,
+    CONF_LONGITUDE,
+    CONF_NAME,
+    CONF_SCAN_INTERVAL,
+)
+from homeassistant.components.air_quality import (
+    AirQualityEntity,
+    ATTR_PM_10,
+    ATTR_PM_2_5,
+)
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.util import Throttle
+
+from .const import CONF_LANGUAGE, DOMAIN, NO_AIRLY_SENSORS
+
+_LOGGER = logging.getLogger(__name__)
+
+ATTRIBUTION = "Data provided by Airly"
+
+ATTR_CAQI = "CAQI"
+ATTR_CAQI_ADVICE = "advice"
+ATTR_CAQI_DESCRIPTION = "DESCRIPTION"
+ATTR_CAQI_LEVEL = "air quality index level"
```

Comment:
Please use lowercase snake_case keys for state attributes.
```suggestion
ATTR_CAQI_LEVEL = "air_quality_index_level"
```

Reply:
We already have `ATTR_AQI = "air_quality_index"` in the base air quality component. Is `ATTR_CAQI_LEVEL` something else? Make sure to use different constants for the api data keys and the state attribute keys. We don't want to couple those tightly.

Reply:
Airly sends `CAQI` as a numeric value and `CAQI_LEVEL` as a verbal description, e.g. "very low".